newIDE/app/src/UI/KeyboardShortcuts/index.js

```diff
@@ -143,16 +143,31 @@ export default class KeyboardShortcuts {
   };
 
   _updateModifiersFromEvent = (evt: KeyboardEvent | DragEvent) => {
+    const hasModifierChanged =
+      this._metaPressed !== evt.metaKey ||
+      this._altPressed !== evt.altKey ||
+      this._ctrlPressed !== evt.ctrlKey ||
+      this._shiftPressed !== evt.shiftKey;
+
     this._metaPressed = evt.metaKey;
     this._altPressed = evt.altKey;
     this._ctrlPressed = evt.ctrlKey;
     this._shiftPressed = evt.shiftKey;
+
+    return hasModifierChanged;
   };
 
   _updateSpecialKeysStatus = (evt: KeyboardEvent, isDown: boolean) => {
     if (evt.which === SPACE_KEY) {
+      // Prevent scrolling in Physics2 property editor.
```

Comment:
I feel like this comment should be "Prevent scrolling on browsers when listening on key events and space is used (to pan an image preview for instance)"